DSharpPlus/Entities/Emoji/DiscordEmoji.EmojiUtils.cs

```diff
@@ -18,15 +18,17 @@ static DiscordEmoji()
     {
         #region Generated Emoji Map
         // Generated by Discord Emoji Map generator by Emzi0767
+        // Normally generated from https://static.emzi0767.com/misc/discordEmojiMap.min.json but Emzi's tool is currently broken as of 2023-12-18T03:25:08.398+00:00
         // Generated from:
-        //   version           2023-02-14T20:04:19.610+00:00
-        //   url               https://static.emzi0767.com/misc/discordEmojiMap.min.json
-        //   definition count  3,650
+        //   version           2023-12-18T03:25:08.398+00:00
+        //   url               http://localhost/discordEmojiMap-canary.min.json
```

Comment:
Where'd you pull the tool from and can we have access to it in some form

Reply:
This comment has since been answered in DM's: https://discord.com/channels/@me/963957084452442233/1186172655859146783